product-mini/platforms/windows/wasi-filtered-tests.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Open to suggestions on the naming/location of this file. Not sure if it fits here